internal/proposals/logs/unified-approach-to-logging-levels.md

```diff
@@ -0,0 +1,66 @@
+# Unified approach to logging levels
+
+We need a consistent way to put the logs into the specific logging severity levels. We also should have a default severity level on each environment such as DEV, STAGE and PROD.
+It also must be possible to easily set a more verbose logging level of each component within the Kyma ecosystem while troubleshooting.
+
+We have some "external" components too (not implemented by the Kyma team) and we need an approach on how to change the logging level in them. For debugging reasons the level could be changed without any code and charts changes just with the in-cluster resource change.
+
+## What to put into each severity level
+
+As we decided to use Zap library as the logger I'll describe all the needed levels from this particular one:
+
+- `FATAL` - a component reached a catastrophic error and cannot go further,
+- `ERROR` - errors that break something, so request failures, reading memory failures, and so on,
+- `WARN` - things that do not break anything but suggest that potential setup should be changed, so resources are running low, optional config cannot be fetched, a request failed but will be retried, a connection is lost but will be reacquired, and so on,
+- `INFO` - requests handling, operating on resources, successes, information about operations in progress, and so on,
+- `DEBUG` - options with which the component is deployed, requests' paths, and other information that is useful while making sure that everything works as expected.
+
+No other severity will be used as it is not needed.
+
+## Default severity levels for the environments
+
+To be consistent, we need to agree on what default level components will log in each environment:
+
+- PROD: `WARN` - this is a minimal severity needed for effective debugging of issues,
+- STAGE: `WARN` - this is a minimal severity needed for effective debugging of issues. It'd also be a good indicator on how the logs will look like on PROD environment,
```

Comment:
```suggestion
- STAGE: `WARN` - this is a minimal severity needed for effective debugging of issues. It would also be a good indicator on how the logs will look like on PROD environment,
```